tests/utils.go

```diff
@@ -1417,6 +1416,48 @@ func RunOcCommand(args ...string) (string, error) {
 	return string(stdOutErrBytes), nil
 }
 
+func SkipIfNoVirtctl() {
+	if KubeVirtVirtctlPath == "" {
+		var err error
+		KubeVirtVirtctlPath, err = exec.LookPath("virtctl")
+		if err != nil {
+			Skip("Skip test that requires virtctl binary")
+		}
+	}
+}
+
+func RunVirtctlCommand(args ...string) (string, error) {
```

Comment:
Can't you use `NewRepeatableVirtctlCommand` and `NewVirtctlCommand` which are also in this file?

Reply:
`NewRepeatableVirtctlCommand` and `NewVirtctlCommand` should be good enough, I didn't notice it.